tests/benchmarks/android_jankbench.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python
+
+import os
+
+# The workload class MUST be loaded before the LisaBenchmark
+from android import Workload
+from android import LisaBenchmark
+
+from devlib.exception import TargetError
+
+class Jankbench(LisaBenchmark):
+
+    bm_conf = {
+
+        # Target platform and board
+        "platform"      : 'android',
+
+        # Define devlib modules to load
+        "modules"     : [
+            'cpufreq',
+        ],
+
+        # FTrace events to collect for all the tests configuration which have
+        # the "ftrace" flag enabled
+        "ftrace"  : {
+            "events" : [
+                "sched_switch",
+                "sched_overutilized",
+                "sched_contrib_scale_f",
+                "sched_load_avg_cpu",
+                "sched_load_avg_task",
+                "sched_tune_tasks_update",
+                "sched_boost_cpu",
+                "sched_boost_task",
+                "sched_energy_diff",
+                "cpu_frequency",
+                "cpu_idle",
+                "cpu_capacity",
+            ],
+            "buffsize" : 10 * 1024,
+        },
+
+        # Default EnergyMeter Configuration
+		"emeter" : {
+			"instrument" : "acme",
+			"channel_map" : {
+				"Device0" : 0,
+			}
+		},
+
+        # Tools required by the experiments
+        "tools"   : [ 'trace-cmd' ],
+
+		# Default results folder
+		"results_dir" : "AndroidJankbench",
+
+    }
+
+	# Android Workload to run
+    bm_kind = 'Jankbench'
+
+    # Default test configuration
+    bm_params = {
+        'test_name'  : 'list_view',
+        'iterations' : 3,
+    }
+
+	# Default products to be collected
+    bm_collect = 'ftrace energy'
+
+    def benchmarkInit(self):
+        self.setupWorkload()
+        return self.setupGovernor()
+
+    def benchmarkFinalize(self):
+		pass
+
+    def __init__(self, governor, test, iterations):
+        self.governor = governor
+        self.test = test
+        self.iterations = iterations
+        super(Jankbench, self).__init__()
+
+    def setupWorkload(self):
+        # Create a results folder for each "governor/test"
+        self.out_dir = os.path.join(self.te.res_dir, governor, test)
+        try:
+                os.stat(self.out_dir)
+        except:
+                os.makedirs(self.out_dir)
+        # Setup workload parameters
+        self.bm_params['test_name'] = test
```

Comment:
Agree... again it was there to be more like an example of the public API... but it does not make sense to have this redundant code. Let's remove the defaults.